Lab5/ScannerIntegration/Parser.py

```diff
@@ -0,0 +1,233 @@
+from Lab5.ScannerIntegration.Grammar import Grammar

```

Comment:
They overlap with word, pdf and other files from the main package. 
Grammar and Parser should be in a separate package.